modules/nw-ingress-reencrypt-route-custom-cert.adoc

```diff
@@ -0,0 +1,58 @@
+// This is included in the following assemblies:
+//
+// networking/routes/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="creating-re-encrypt-route-with-custom-certificate_{context}"]
+= Creating a route using the destination CA certificate in the Ingress annotation
+
+The `route.openshift.io/destination-ca-certificate-secret` annotation can be used on an Ingress object to define an route with a custom certificate (CA).
+
+.Prerequisites
+* You must have a certificate/key pair in PEM-encoded files, where the certificate is valid for the route host.
```

Comment:
Maybe `certificate/key pair` -> `certificate or key pair` instead. Up to you!

Reply:
certificate/key pair is common phrasing and most accurate here, so I will retain it. Thanks!